go/enclave/enclave.go

```diff
@@ -1344,38 +1344,60 @@ func (e *enclaveImpl) GetReceiptsByAddress(encryptedParams common.EncryptedParam
 		return responses.AsPlaintextError(fmt.Errorf("unable to decode eth_getStorageAt params - %w", err)), nil
 	}
 
-	// Parameters are [ViewingKey, Address, TBD, TBD]
+	// Parameters are [ViewingKey, PrivateCustomQueryHeader, PrivateCustomQueryArgs, null]
```

Comment:
Why is required number of parameters == 4 (with last being null). Is this because we reuse getStorageAt?

Reply:
Yep. The length of the params slice is 4, even though the 4th position is null. 
This avoids a few problems we had in past were we would receive params slices with a length of 3 and then had to implement weird logic to accommodate for it.